ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -8420,7 +8428,7 @@ SDHC:
   - leaving_dehumidifier_temperature_sensor
   - leaving_dehumidifier_temperature_setpoint
   implements:
-  - CONTROL
+  - CONTROL  
```

Comment:
please remove the trailing space here

Reply:
@shambergoldstein Is the trailing space still there?

Reply:
yes, if you are unable to remove it dont worry about it and I can take it out after

Reply:
Yes, because I am not seeing it for some reason or unable to spot it. Thanks